README.md

```diff
@@ -1,24 +1,221 @@
+
+* Ruby on Rails.
+* PostgreSQL.
+* RSpec.
+* Rubocop.
+* Stylelint.
+
+<br>
+
+---
+
+<!-- Features -->
+
+### Key Features
+
+- [ ] Create, edit, and delete posts.
+- [ ] Comment system to interact with posts.
+- [ ] Like and dislike posts.
+- [ ] User authentication and authorization.
+- [ ] Responsive design.
+
+Please note: Each feature will be marked with an "x" when it is fully implemented and operational.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+---
+
+### 🔭 Upcoming Features <a name="future-project"></a>
+
+- [ ] Search functionality for locating specific articles.
+- [ ] User profiles with customizable avatars.
+- [ ] Article categories and tagging.
+- [ ] Rich text editor for formatting articles.
+- [ ] Social media sharing options for articles.
+- [ ] API endpoints for external integrations.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+---
+
+
+## 🚀 Live Demo <a name="live-demo"></a>
+<br>
+
+- Unfortunately, there is no live demo available for this project at the moment.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+---
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a id="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+In order to run this project you need to have:
+
+- Installed Ruby and Ruby on Rails.
+- If you're using Windows, have installed WSL.
+- If you're using MacOS, an option is using UTM.
+
+### Setup
+To clone this repository to your desired folder, run the following command: <br>
+
+```
+git clone git@github.com:Ruthmy/blog-app.git
+```
+
+Navigate to the cloned repository and run the following command to install the necessary gems:
+```
+cd blog-app
+bundle install
+```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+### Usage
+Initialize it with
+```
+rails server
+```
+Then open `http://localhost:3000` in your browser.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+### Test
+To run the tests try the following command:
+```
+rspec spec
+```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+---
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a id="authors"></a>
+
+👤 **Ruth Abreu**
+
+- GitHub: [@Ruthmy](https://github.com/Ruthmy)
+- Twitter [@rury_exe](https://twitter.com/rury_exe)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/ruth-abreu)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+---
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a id="future-features"></a>
+* Save the information in a PostgreSQL database.
+* Let the user to remove and modify the information.
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+---
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a id="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```

Comment:
Great job @Ruthmy 🤝

- [ ] Kindly fix the broken issues link to make your readme file more professional.
![issues](https://github.com/Ruthmy/blog-app/assets/63917725/56e1af69-8f96-41a1-955a-35bfb0ff28af)

Reply:
### Hi @1GORDON 
### Thank you very much for your feedback, I will fix it right away ✨ ✨